src/helpers/resizedImageCache.ts

```diff
@@ -0,0 +1,29 @@
+import ResizedImages from '../models/ResizedImages'
+
+export async function getCachedResize(
+  cid: string,
+  height?: number,
+  width?: number
+): Promise<string | null> {
+  const resizedImage = await ResizedImages.findOne({
+    cid,
+    height: height ? height : null,
+    width: width ? width : null,
+  })
+  if (!resizedImage) return null
+  return resizedImage.resizedCid
+}
+
+export async function setCachedResize(
+  cid: string,
+  height: number | undefined,
```

Comment:
Let's do `height?: number` like in `getCachedResize` above

Reply:
We can't because `resizedCid` is not optional.